alma/lib/Utils/LocaleHelper.php

```diff
@@ -108,4 +109,55 @@ public static function thousandSeparator()
             return $symbols->getGroup();
         }
     }
+
+    /**
+     * Get translation by file iso if exist
+     *
+     * @param array $string
+     * @param string $source
+     * @param string $locale
+     *
+     * @return array
+     *
+     * @see AdminTranslationsController::getModuleTranslation
+     */
+    public static function getModuleTranslation(
```

Comment:
You can (should) test this method using varying `_PS_MODULE_DIR_` with test translations `.php` files.

You can use TDD here for the `not translated` website issue ❣️

Reply:
I would like to talk about it